src/Conversion/KrnlToLLVM/KrnlToLLVM.cpp

```diff
@@ -163,6 +163,41 @@ static FlatSymbolRefAttr getOrInsertMemcpy(
   return SymbolRefAttr::get(context, "llvm.memcpy.p0i8.p0i8.i64");
 }
 
+static FlatSymbolRefAttr getOrInsertRandomNormal(
+    PatternRewriter &rewriter, ModuleOp module, mlir::Type inType) {
+  auto *context = module.getContext();
```

Comment:
[Suggestion]: Using `auto` is convenient but does not convey the static type to readers. The LLVM code style rule encourage to not use `auto` when the type is obvious:
```
Don’t “almost always” use auto, but do use auto with initializers like cast<Foo>(...) or other places where the type is already obvious from the context"

Reply:
personally agree with minimizing auto.